pkg/operator/ceph/cluster/controller.go

```diff
@@ -130,6 +131,11 @@ func add(opManagerContext context.Context, mgr manager.Manager, r reconcile.Reco
 	}
 	logger.Info("successfully started")
 
+	err = addonsv1alpha1.AddToScheme(mgr.GetScheme())
+	if err != nil {
+		logger.Warningf("failed to add csi-addons to scheme: %v", err)
```

Comment:
Need to return an error?

Reply:
we don't want to return an error for the whole fencing right? so that it doesn't block existing reconcile?

Reply:
Will this fail if the CSI addons CRDs are not installed? If so, then certainly don't want to return an error

Reply:
I quickly I did not see any warning with that string in operator logs, so adding the `Errof`

Reply:
If we don't expect it to fail even when the CRDs are not there, then we always want it to succeed. So I'm suggesting to return an err so that the operator will retry.

Reply:
okay